test/new-e2e/examples/vm_fakeintake_test.go

```diff
@@ -0,0 +1,79 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package examples
+
+import (
+	_ "embed"
+	"errors"
+	"time"
+
+	"testing"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/utils/e2e"
+	ec2vm "github.com/DataDog/test-infra-definitions/aws/scenarios/vm/ec2VM"
+	"github.com/DataDog/test-infra-definitions/aws/scenarios/vm/os"
+	commonos "github.com/DataDog/test-infra-definitions/common/os"
+	"github.com/cenkalti/backoff/v4"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type vmFakeintakeSuite struct {
+	e2e.Suite[e2e.AgentEnv]
+}
+
+func TestE2EVMFakeintakeSuite(t *testing.T) {
+	e2e.Run(t, &vmFakeintakeSuite{}, e2e.AgentStackDef([]e2e.Ec2VMOption{ec2vm.WithArch(os.WindowsOS, commonos.AMD64Arch)}))
+}
+
+func (s *vmFakeintakeSuite) TestVM() {
+	output := s.Env().VM.Execute("ls")
+	require.NotEmpty(s.T(), output)
+}
+
+func (s *vmFakeintakeSuite) TestAgent() {
+	output := s.Env().Agent.Status()
+
+	require.Contains(s.T(), output.Content, "Getting the status from the agent")
```

Comment:
This can be flaky if the Agent starts too quickly.

Reply:
Need to prepend with a wait for agent ready